odoo/http.py

```diff
@@ -507,8 +507,16 @@ def decorator(f):
             else:
                 routes = [route]
             routing['routes'] = routes
+
         @functools.wraps(f)
         def response_wrap(*args, **kw):
+            # if controller cannot be called with extra args (utm, debug, ...), call endpoint ignoring them
+            spec = inspect.getargspec(f)
+            if not spec.keywords:
+                ignored = ['<%s=%s>' % (k, kw.pop(k)) for k in list(kw) if k not in spec.args]
+                if ignored:
+                    _logger.info("<function %s.%s> called ignoring args %s" % (f.__module__, f.__name__, ', '.join(ignored)))
```

Comment:
Got similar on v13
`2019-10-07 17:14:01,700 8935 INFO 13_sec odoo.http: <function odoo.addons.web.controllers.main.call> called ignoring args <kwargs={'context': {'lang': 'en_US'}}>`

Not sure it's relevant